### DISCUSSION TITLE
Feature request: Disable sidebar in new UI when starting just one task

### DISCUSSION THREAD

**Discussion by MickL** [vercel/turborepo#8430] [Ideas]
### Goals

When starting just one task (e.g. with --filter="...") there is no need for the left sidebar. In this case the whole terminal UI can be deactivated.

As a workaround for now when starting just one task at least the task should be interactive immediately without the need to enter it with the `enter` key. It is very scary to press enter when Drizzle asks me if I really want to do a certain action.

### Background

A common task single task would be `drizzle push` for example.

### Proposal

-

**Comment by anthonyshew** [comment]
Duplicate of #8351
